tools/publish-packages.js

```diff
@@ -20,21 +21,23 @@ const defaultOptions = {
   exclude: null,
 };
 
+const ROOT_DIR = path.dirname(__dirname);
+
+// sed command differs between platforms
+const SED = os.platform() === 'linux' ? 'sed' : 'gsed';
+
 // configs for other packages that are not unimodules
 const otherPackages = [
   {
     libName: 'expo',
-    dir: `${process.env.EXPO_UNIVERSE_DIR}/exponent/packages/expo`,
+    dir: `${ROOT_DIR}/packages/expo`,
```

Comment:
Are you sure `${ROOT_DIR}/packages/expo` is a valid `dir`? Isn't `ROOT_DIR = ~/…/expo/tools`?